README.md

```diff
@@ -1,4 +1,4 @@
-# Getting Started with Create React App
+# math-magicians-calculator
```

Comment:
- [ ] Please update the readme file add the project description etc...., please don't make the last thing to change.